src/app/EventManagement.h

```diff
@@ -449,6 +451,19 @@ class EventManagement
      */
     EventNumber GetFirstEventNumber(PriorityLevel aPriority);
 
+    /**
+     * @brief
+     *   IsValid returns whether the EventManagement instance is valid
+     */
+    bool IsValid(void) { return EventManagementStates::Shutdown != mState; };
+
+    /**
+     *  Logger would save last logged event number for each logger buffer into schedule event number array
+     *
```

Comment:
nit: remove some of the empty whitespace here.

Reply:
Done, thanks